Wendy/homework6_part1.js

```diff
@@ -0,0 +1,50 @@
+
+//HOMEWORK PART A//
+function Ctor(a,b){
+	this.a = 0;
+	this.b = 1;
+}
+
+var obj1 = new Ctor(3,1)
+var obj2 = new Ctor(1,22)
+
+
+var obj3 = {};
+Ctor.call(obj3);
+
+obj1.c = 3;
+3
+obj1.c
+3
+obj2.c
+undefined
+
+obj1.constructor.prototype.d = 3;
+3
+obj1.d
+3
+obj2.d
+3
+obj3.d
+undefined
+//If I create an obj3 = new Ctor than the value of obj3.d is 3.
+//Adding to the prototype then alters the original template-- the original Ctor.
+//What's interesting is that if I experiment and write:
+//Ctor.constructor.d=4, then type in obj1.d, the values is still 3.
+//don't mess with the protoptype unless you really want it to be altered forever.
+
```

Comment:
You're correct in saying that, if you create an obj3 = new Ctor then the value of obj3.d is 3. Furthermore, the difference in the above obj1 and obj2 are that they are both made with the "new", and are therefore automatically linked to the Ctor.prototype, whereas obj3 is made implicitly, so its proto is Object.prototype, and it will not inherit from Ctor.prototype.